desktop/cmp/grouping/GroupingChooser.ts

```diff
@@ -34,14 +33,14 @@ export interface GroupingChooserProps extends ButtonProps<GroupingChooserModel>
     /** Title for popover (default "GROUP BY") or null to suppress. */
     popoverTitle?: string;
 
-    /** Width in pixels of the popover menu itself. */
+    /**
+     * Width in pixels of the popover menu itself.
+     * If unspecified, will default based on whether favorites are enabled.
+     */
```

Comment:
A bit on the fence if this still makes sense to allow customization - I think so, but currently it will be a 50/50 split between value and fave editing (unless developer gets into the CSS, which they could do)